### ISSUE TITLE
focus_follows_pointer fails on drag'n drop

### ISSUE THREAD

**Issue by taz4hvn** [baskerville/bspwm#718]
When I drag a file from, say, my window manager to my text editor to open it, file is opened as expected but focus stays on the file manager while obviously pointer is on text editor.
This is really annoying to me since I will start typing the edited file but I'm actually sending keystrokes to the file manager !
Browsing the opened file, opening menu or moving pointer in the editor doesn't give focus to it, I have to leave the editor region, focus another window and go back to give the focus to the editor.
Same happens if I leave a window with a menu opened, focus doesn't follow mouse, I have to go back, close the menu, and finally return to the previously aimed window. Is there something I can do ?
BTW I'm using the bspwm-git  1:0.9.3.r18.g835ab36-1 on manjaro.

**Comment by taz4hvn** [comment]
I made some progress, I can focus with any button now, I had in my bspcwmrc:
bspc config click_to_focus true
instead of
bspc config click_to_focus any
but still can't have focus to follow only by dragging.

**Comment by jallbrit** [comment]
@taz4hvn are you still experiencing this bug?

**Comment by taz4hvn** [comment]
Yes. Opening a menu in any window prevents any focus change until menu is dismissed. I learnt to do with it.
